### ISSUE TITLE
Cannot enter text in REPL-MODE

### ISSUE THREAD

**Issue by aecepoglu** [atlas-engineer/nyxt#963]
After switching into repl-mode with `lisp-repl`, none of the text I enter shows up in the prompt. Pressing ENTER key produces the error in the stdout:

```
In RETURN-INPUT: There is no applicable method for the generic function #<STANDARD-GENERIC_FUNCTION NYXT:INPUT-BUFFER (2)> when called with arguments (NIL).
```
 
**Precise recipe to reproduce the issue**

1. start nyxt
2. `C-space` and `lisp-repl`
3. Type on the keyboard
4. I expect to see some text in the repl prompt but nothing shows up

**Information**

- Nyxt version 2-pre-release-1-171-gad06b8bc
- Installed on archlinux via aur

**Comment by jmercouris** [comment]
Thanks for the report, it seems like we broke it in a recent commit!

**Comment by jmercouris** [comment]
Fixed in commit: a2fd83b4

please let me know if you have any other problems! Thank you!

**Comment by aecepoglu** [comment]
seems to work, cheers.
